content/90.back-matter.md

```diff
@@ -2,3 +2,344 @@
+[@tag:Codella2016_ensemble_melanoma]: arxiv:1610.04662
+[@tag:Consortium2012_encode]: doi:10.1038/nature11247
+[@tag:CudNN]: arxiv:1410.0759
+[@tag:Dahl2014_multi_qsar]: arxiv:1406.1231
+[@tag:Darst2018]: doi:10.1186/s12863-018-0646-3
+[@tag:Dean2012_nips_downpour]: url:http://research.google.com/archive/large_deep_networks_nips2012.html
+[@tag:DeepChem]: url:https://github.com/deepchem/deepchem
+[@tag:Deming2016_genetic]: arxiv:1605.07156
+[@tag:Ding]: doi:10.1186/s12859-015-0753-3
+[@tag:Ditzler]: doi:10.1186/s12859-015-0793-8
+[@tag:Ditzler2]: doi:10.1109/TNNLS.2014.2320415
+[@tag:Ditzler3]: doi:10.1109/TNB.2015.2461219
+[@tag:Dhungel2015_struct_pred_mamm]: doi:10.1007/978-3-319-24553-9_74
+[@tag:Dhungel2016_mamm]: doi:10.1007/978-3-319-46723-8_13
+[@tag:Dhungel2017_mamm_min_interv]: doi:10.1016/j.media.2017.01.009
+[@tag:Dream_tf_binding]: url:https://www.synapse.org/#!Synapse:syn6131484/wiki/402026
+[@tag:Dragonn]: url:http://kundajelab.github.io/dragonn/
+[@tag:Duvenaud2015_graph_conv]: url:http://papers.nips.cc/paper/5954-convolutional-networks-on-graphs-for-learning-molecular-fingerprints
+[@tag:Edwards2015_growing_pains]: doi:10.1145/2771283
+[@tag:Ehran2009_visualizing]: url:http://www.iro.umontreal.ca/~lisa/publications2/index.php/publications/show/247
+[@tag:Elephas]: url:https://github.com/maxpumperla/elephas
+[@tag:Errington2014_reproducibility]: doi:10.7554/eLife.04333
+[@tag:Eser2016_fiddle]: doi:10.1101/081380
+[@tag:Esfahani2016_melanoma]: doi:10.1109/EMBC.2016.7590963
+[@tag:Essinger2010_taxonomic]: doi:10.1109/IJCNN.2010.5596644
+[@tag:Esteva2017_skin_cancer_nature]: doi:10.1038/nature21056
+[@tag:Faruqi]: url:http://alifar76.github.io/sklearn-metrics/
+[@tag:Feinberg2018]: doi:10.1056/NEJMra1402513
+[@tag:Finnegan2017_maximum]: doi:10.1101/105957
+[@tag:Fong2017_perturb]: doi:10.1109/ICCV.2017.371
+[@tag:Fraga2005]: doi:10.1073/pnas.0500398102
+[@tag:Fu2019]: doi:10.1109/TCBB.2019.2909237
+[@tag:Gal2015_dropout]: arxiv:1506.02142
+[@tag:Gaublomme2015_th17]: doi:10.1016/j.cell.2015.11.009
+[@tag:Gargeya2017_dr]: doi:10.1016/j.ophtha.2017.02.008
+[@tag:Gawad2016_singlecell]: doi:10.1038/nrg.2015.16
+[@tag:Geras2017_multiview_mamm]: doi:10.1038/nrg.2015.16
+[@tag:Gerstein2016_scaling]: doi:10.1186/s13059-016-0917-0
+[@tag:Ghandi2014_enhanced]: doi:10.1371/journal.pcbi.1003711
+[@tag:Ghosh1992_sequence]: doi:10.1117/12.140112
+[@tag:Glorot2011_domain]: url:http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.231.3442
+[@tag:Goldsborough2017_cytogan]: doi:10.1101/227645
+[@tag:Gomezb2016_automatic]: arxiv:1610.02415v1
+[@tag:Graphlab]: doi:10.14778/2212351.2212354
+[@tag:Groop1986_islet]: doi:10.2337/diab.35.2.237
+[@tag:Gultepe2014_sepsis]: doi:10.1136/amiajnl-2013-001815
+[@tag:Gulshan2016_dt]: doi:10.1001/jama.2016.17216
+[@tag:Gupta2015_exprs_yeast]: doi:10.1101/031906
+[@tag:Gupta2015_prec]: arxiv:1502.02551
+[@tag:Guetterman]: url:http://www.fasebj.org/content/30/1_Supplement/406.3
```

Comment:
If we freeze references post-merge, we need to add a manual reference for this first.  We're adding a broken DOI link https://doi.org/10.1096/fasebj.30.1_supplement.406.3